app/routes/task_routes.py

```diff
@@ -1 +1,126 @@
-from flask import Blueprint
\ No newline at end of file
+import requests
+from flask import Blueprint, abort, make_response, request
+from ..db import db
+from app.models.task import Task
+from datetime import datetime
+import os
+from dotenv import load_dotenv
+from .route_utilities import validate_model
+
+load_dotenv()
+
+bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+
+@bp.post("")
+def create_task():
+    request_body = request.get_json()
+    if "title" not in request_body or "description" not in request_body:
+        abort(make_response({"details": "Invalid data"}, 400))
+
+    completed_at = request_body.get("completed_at", None)
+            
+    new_task = Task.from_dict(request_body)
+    db.session.add(new_task)
+    db.session.commit()
+
+    response = {
+        "task": new_task.to_dict()
+    }
+    return response, 201
```

Comment:
Good job! Good error handling here. Try using a try/except block over if statements as that is more in line with best practices.